traffic_portal/app/src/modules/private/roles/list/index.js

```diff
@@ -28,7 +28,7 @@ module.exports = angular.module('trafficPortal.private.roles.list', [])
 						controller: 'TableRolesController',
 						resolve: {
 							roles: function(roleService) {
-								return roleService.getRoles({ orderby: 'priv_level DESC' });
+								return roleService.getRoles({ orderby: 'privLevel', orderbydown : '' });
```

Comment:
for some reason a query param with no value feels odd to me (although it might be technically ok). what about a query parameter called `sortOrder` or something with 2 values: asc or desc and if sortOrder is not provided with an orderby, then asc is the default.